client/disputes/index.tsx

```diff
@@ -530,6 +433,14 @@ export const DisputesList = (): JSX.Element => {
 		<Page>
 			<TestModeNotice currentPage="disputes" />
 			<DisputesFilters storeCurrencies={ storeCurrencies } />
+			{ isExportInProgress && (
```

Comment:
New approach has been tried in #10211  based on design review. Please have a look, test and once that is approved and merged, we can use that here.